zulipterminal/core.py

```diff
@@ -523,6 +523,22 @@ def stream_muting_confirmation_popup(
         mute_this_stream = partial(self.model.toggle_stream_muted_status, stream_id)
         self.loop.widget = PopUpConfirmationView(self, question, mute_this_stream)
 
+    def exit_compose_confirmation_popup(self) -> None:
+        question = urwid.Text(
+            (
+                "bold",
+                "Please confirm that you wish to exit the compose box. "
+                "(You can save the message as a draft upon returning to compose)",
```

Comment:
The bottom line wraps partially for me to be on the same line as the first - I think it makes sense to have these at least on separate lines.